x-pack/legacy/plugins/siem/cypress/integration/smoke_tests/timeline/toggle_column.spec.ts

```diff
@@ -68,24 +73,24 @@ describe('toggle column in timeline', () => {
     cy.get(`[data-test-subj="timeline"] [data-test-subj="header-text-${idField}"]`).should('exist');
   });
 
-  it('adds the _id field to the timeline via drag and drop', () => {
+  it('adds the _index field to the timeline via drag and drop', () => {
```

Comment:
It appears this change was necessary because the `_id` column, which was added in a previous test, is not cleared by `clearTimeline()`, per the screenshot below:

![_id_field_in_timeline](https://user-images.githubusercontent.com/4459398/72182857-9d505a80-33a9-11ea-9302-71cf180e4adc.png)

The Fields Browser `Reset Fields` button restores all default fields in the timeline, and there's an example of this feature being tested in the `it('resets all fields in the timeline when Reset Fields is clicked')` test, in `x-pack/legacy/plugins/siem/cypress/integration/smoke_tests/fields_browser/fields_browser.spec.ts`, but rather than adding this as an additional step in `clearTimeline()`, I think a slightly more bulletproof implementation of `clearTimeline()` would be to invoke the `Create New Timeline` menu item shown in the screenshot below, which will reset all of the state currently cleared by `clearTimeline()`, plus much more, including the column state:

![create_new_timeline](https://user-images.githubusercontent.com/4459398/72182484-c9b7a700-33a8-11ea-879c-dd42e8d854dd.png)

Please consider replacing the implementation of `clearTimeline()` to use `Create New Timeline`, and reverting the changes to this test.